### PR TITLE
Add learning goals for starter tutorials

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tutorials/configure/build-a-mock-robot.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ date: "2022-10-11"
 cost: "0"
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will know how to use fake components to prototype if you don't have (the right) components at hand. You will also know how to swap out your fake components for real components when you get them.
```
Comment (andf-viam) [inline docs/tutorials/configure/build-a-mock-robot.md:27]: Nit: slightly more straightforward phrasing, avoiding parens:

```suggestion
After following this tutorial, you will know how to use fake components to prototype a project without needing any hardware components. You will also know how to swap out your fake components for real components when you are ready to do so.
```

Goal: 'know how to swap fake for real' is a great addition, we don't cover that anywhere and it is what likely every Mock user will need once succeeding in the tut. 👍

File: docs/tutorials/get-started/blink-an-led.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ no_list: true
 weight: 1
 ---
 
+<!-- LEARNING GOALS:
+After following this tutorial, you will understand when to use a board component and be able to configure a board component for your SBC and control the component using the Viam app and the Viam SDKs.
```
Comment (andf-viam) [inline docs/tutorials/get-started/blink-an-led.md:28]: Supernit: phrasing simplification:

```suggestion
After following this tutorial, you will understand when to use a board component and be able to configure a board component for your SBC and control it using the Viam app and the Viam SDKs.
```

File: docs/tutorials/get-started/servo-mousemover.md [hunk 3]
```diff
@@ -21,6 +21,11 @@ cost: 80
 no_list: true
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will know when to use a motor component and be able to configure a servo component and control the component using the Viam app the Viam SDKs.
```
Comment (andf-viam) [inline docs/tutorials/get-started/servo-mousemover.md:25]: Supernit: phrasing:

```suggestion
After following this tutorial, you will know when to use a motor component and be able to configure a servo component and control it using the Viam app the Viam SDKs.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md [hunk 4]
```diff
@@ -17,6 +17,13 @@ no_list: true
 # SMES: Olivia Miller, Sierra Guequierre, Hazal Mestci
 ---
 
+<!-- LEARNING GOALS:
+After this tutorial you understand viam's modular platform and can build a machine from start to finish. You will be able to create machines using builtin and registry resources and write code with the SDKs to operate your machine
```
Comment (andf-viam) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:21]: Same nit: avoid `platform` & some phrasing:

```suggestion
After this tutorial you will understand Viam's modular resources, and can build a machine from start to finish. You will be able to create machines using built-in resources as well as resources from the VIam registry, and can write code with the SDKs to operate your machine.
```

**Review by andf-viam — APPROVED** [review]
LGTM with some light phrasing suggestions. Thank you!

File: docs/tutorials/projects/make-a-plant-watering-robot.md (hunk 4 above)
Comment (npentrel) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:21]:
```suggestion
After this tutorial you will understand what modules are and when to use them, and be able to build a machine from start to finish. You will be able to create machines using built-in resources as well as resources from the Viam registry, and can write code with the SDKs to operate your machine.
```

**Pull request merged**